src/shared/common/components/side-bar.jsx

```diff
@@ -59,19 +64,54 @@ class SideBar extends Component {
         );
     }
 
+    updateTabs() {
+        const { page, dispatch } = this.props;
+        if (page && page.params && page.params.caseId) {
+            updateCaseConfig(page.params.caseId, dispatch);
+        }
+    }
+
+    updateActiveTab() {
+        const { caseConfig } = this.props;
+
+        if (caseConfig && !this.state.active) {
+            this.setState({ active: this.getLeftMostTab(caseConfig.tabs) });
+        }
+    }
+
+    getLeftMostTab(caseTabs) {
+        if (caseTabs.includes('documents')) {
+            return 'DOCUMENTS';
+        }
+        if (caseTabs.includes('summary')) {
+            return 'SUMMARY';
+        }
+        if (caseTabs.includes('timeline')) {
+            return 'TIMELINE';
+        }
+        if (caseTabs.includes('people')) {
+            return 'PEOPLE';
+        }
+        if (caseTabs.includes('actions')) {
+            return 'CASE_ACTIONS';
+        }
+        if (caseTabs.includes('ex_gratia')) {
+            return 'EX_GRATIA';
+        }
+    }
```

Comment:
I think there is an opportunity here to make the order defined by config with a sort order, then here we take the first item depending on the sort order, rather than relying on the order hard coded in a method here.

Happy to talk through,

Cheers,
Rob